assets/scripts/streets/StreetName.jsx

```diff
@@ -116,6 +103,7 @@ export default class StreetName extends React.Component {
 }
 
 StreetName.propTypes = {
+  id: React.PropTypes.string,
   allowEditing: React.PropTypes.bool,
   street: React.PropTypes.any,
   handleResize: React.PropTypes.func
```

Comment:
I pushed a commit that adds a defaultProp instead. I think that's a tad bit cleaner than making the other parent components care about the no-op. What do you think?